packages/react-native/scripts/react_native_pods.rb

```diff
@@ -246,6 +246,7 @@ def react_native_post_install(
   ReactNativePodsUtils.set_node_modules_user_settings(installer, react_native_path)
   ReactNativePodsUtils.apply_flags_for_fabric(installer, fabric_enabled: fabric_enabled)
   ReactNativePodsUtils.apply_xcode_15_patch(installer)
+  ReactNativePodsUtils.apply_ats_config(installer)
```

Comment:
This should ultimately only be running on the target that the user added the function to, and not all targets in the pbxproj.